gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -3,3 +3,81 @@
  # See related issue: https://github.com/googleapis/gapic-generator-python/issues/2121.
 #}
 {% set rest_async_io_enabled = api.all_library_settings[api.naming.proto_package].python_settings.experimental_features.rest_async_io_enabled %}
+{% extends '_base.py.j2' %}
+
+{% block content %}
+
+from google.api_core import gapic_v1
+
+from typing import Any, Optional
+
+from .rest_base import _Base{{ service.name }}RestTransport
+
+from .base import DEFAULT_CLIENT_INFO as BASE_DEFAULT_CLIENT_INFO
+
+{# TODO (ohmayr): determine the version of rest. aiohttp or google.auth.aio. #}
```

Comment:
If you have tracking bugs (internal or external) for these TODOs, use them in place of your name.

(just being extra-cautious in case things get delayed; I want to maximize our chances of coming back to this promptly)

Reply:
addressed.